src/abstractdataframe/sort.jl

```diff
@@ -355,7 +354,7 @@ end
 
 Return a copy of data frame `df` sorted by column(s) `cols`.
 `cols` can be either a `Symbol` or `Integer` column index, or
-a tuple or vector of such indices.
+a vector of such indices, 'All', `Not`, `Between`, `:`, or `Regex`.
```

Comment:
```suggestion
a vector of such indices, `All`, `Not`, `Between`, `:`, or `Regex`.
```

BTW, is that the standard order in which we list these? The position of `:` isn't very logical I'd say.

Reply:
I will move `:` to the front everywhere in this PR.